### PR TITLE
Move rhsm warning in repo configuring

### PR DESCRIPTION
**Pull request by lzap**
We tell the reader to skip whole chapter, however we only want them to skip first two items on the list involving rhsm. Then the rest still applies.

### CODE REVIEW

File: guides/common/modules/proc_configuring-repositories.adoc [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
 
-ifeval::["{build}" == "foreman"]
-This procedure is only for Katello plug-in and {RHEL}-based operating system users.
-endif::[]
-
 Use this procedure to enable the repositories that are required to install {ProductName}.
 
 .Procedure
+ifeval::["{build}" == "foreman"]
+Skip steps involving `subscription-manager` when not installing on {RHEL} operating system.
```
Comment (spetrosi) [inline guides/common/modules/proc_configuring-repositories.adoc:9]:
```suggestion
This procedure is only for Katello plug-in users. Skip steps involving `subscription-manager` when not installing on the {RHEL} operating system.
```

Comment (spetrosi) [reply guides/common/modules/proc_configuring-repositories.adoc:9]: Return the Katello bit back and add an article for "operating system".

Comment (lzap) [reply guides/common/modules/proc_configuring-repositories.adoc]: > Return the Katello bit back and add an article for "operating system".

I am not following. What article? Sorry :-)

**Comment by spetrosi** [comment]
Thanks @lzap, please rebase to please Travis.

**Comment by spetrosi** [comment]
I was wrong, the links are broken on master and are being fixed now.

**Comment by spetrosi** [comment]
Thank you for contributing, @lzap

**Pull request merged**